base_classes/NXdata.nxdl.xml

```diff
@@ -349,6 +354,21 @@
 			<doc>data label</doc>
 		</attribute>
 	</field>
+	<field name="DATA_errors" type="NX_NUMBER" nameType="any">
+		<doc>
+			Errors (uncertainties) associated with data ``DATA``.
+			Client is responsible for defining the dimensions of the data.
+			The name of this field may be changed to fit the circumstances 
+			but is matched with the *DATA*
+			field with ``_errors`` appended.
```

Comment:
Should we deprecate the DATA_errors field?

Reply:
2018-01-16 telco summary was *Yes*, but indicate in the documentation that the name for an uncertainty might be `DATA_errors`

Reply:
I have changed the wording. This is the only amendment the telco would need to discuss (apart from the original submission, of course).